decidim-proposals/spec/lib/decidim/proposals/import/proposal_creator_spec.rb

```diff
@@ -8,17 +8,17 @@
   let!(:moment) { Time.current }
   let(:data) do
     {
-      id: 1337,
+      :id => 1337,
       "id" => "101",
-      category:,
-      scope:,
+      :category => category,
+      :scope => scope,
       :"title/en" => Faker::Lorem.sentence,
       :"body/en" => Faker::Lorem.paragraph(sentence_count: 3),
```

Comment:
We could use the new syntax if we could use Decidim::Faker::Localized.paragraph  for body element

Reply:
I think it was intentionally mixing the syntax here, as we're [checking out with both syntaxes in this example](https://github.com/decidim/decidim/blob/9b0e22446ca6cedce7945e8388d7630c87c8705b/decidim-proposals/spec/lib/decidim/proposals/import/proposal_creator_spec.rb#L39)

Reply:
Looking to it, i am thinking more that it simulates the CSV header where we have "title/en" or "body/en" keys. I am ok with it.